### ISSUE TITLE
Get corpse name in contract

### ISSUE THREAD

**Issue by Zyles** [esi/esi-issues#1054]
# Feature Request

Need ability to get the corpse name in a contract. Not just "Corpse Male" but the actual player name.

## Use case

Knowing what corpses are in a contract before accepting. Now you have no idea.

## Example return

Character ID or character name for the corpse item.

**Comment by skiedude** [comment]
I don't think this is as much an ESI issue, as it is a game design issue. In game you cannot see the name of the corpse in a contract (confirmed in game). In order to do a corpse sale you either have to use a trusted 3rd party, trade window, or trust the person. There are dozens of these posts on the forums -> sales section.

That being said, maybe ESI can see the meta data of the item that for whatever reason the in game contract cannot display, but this sounds more like a game issue, than an ESI issue.
